modules/virt-about-downward-metrics.adoc

```diff
@@ -0,0 +1,8 @@
+// Module included in the following assemblies:
+//
+// * virt/monitoring/virt-using-downward-metrics.adoc
```

Comment:
Nit - I can't see where this module is located. It's fine to keep this module in this PR if you're not using it, but if it's not in any assemblies, could you delete this one line so it doesn't seem like it's included anywhere?

```suggestion
```